Client/Frontend/Browser/TabsBar/TabBarCell.swift

```diff
@@ -109,12 +111,19 @@ class TabBarCell: UICollectionViewCell {
         }
     }
     
+    func setTheme(with activeTheme: Theme) {
+        backgroundColor = .clear
+        titleLabel.textColor = PrivateBrowsingManager.shared.isPrivateBrowsing ? .white : activeTheme.colors.tints.header
+        closeButton.tintColor = activeTheme.colors.tints.header
+    }
+    
```

Comment:
Probably just easier to make `TabBarCell` `Themable` and use `applyTheme` that all other instances of theming use :)

Reply:
setTheme is removed and Themable protocol is added along side with appleTheme method inside the cell